test/test.js

```diff
@@ -280,3 +281,32 @@ tape('completion detection when final files visited are not tests', t => {
     t.end();
   });
 });
+
+tape('valid source directory (with hashbang tests)', t => {
+  const fixtureDir = path.join(__dirname, 'collateral', 'valid-with-hashbang');
+  const stream = new TestStream(path.join(fixtureDir, 'fake-test262'));
+  const ids = [];
+
+  stream.on('data', makeDataHandler(t, ids, fixtureDir));
+
+  stream.on('error', (error) => {
+    t.ok(error);
+    t.end(error);
+  });
+
+  stream.on('end', () => {
+    const hashbang = '#!whatever\n';
+    const expectedContentDir = path.join(fixtureDir, 'expected-content');
+    const expectedMetadataDir = path.join(fixtureDir, 'expected-metadata');
```

Comment:
This constant is unused, but as noted below, I think much of this code can be removed.